js/components/user/form.vue

```diff
@@ -7,24 +7,44 @@ import User from 'models/user';
 
 export default {
     props: {
+        // Edited user
         user: {
             type: User,
             default: function() {return new User();}
-        }
+        },
+        // Admin property of the editing user (i.e. NOT of the edited one)
+        admin: {
```

Comment:
I is already available with `$root.me.is_admin` in templates and `this.$root.me.is_admin` in Vue code